client/src/App.js

```diff
@@ -1,9 +1,41 @@
+import { useEffect, useState } from 'react';
 import Ohm from './pages/Ohm';
+import ExercisesCategory from './components/ExercisesCategory';
+import BodyParts from './pages/BodyParts';
+import { Route, Switch } from 'react-router-dom';
 
 function App() {
+  const [exercises, setExercises] = useState([]);
+  const [timerExpired, setTimerExpired] = useState(false);
+
+  //Verknüpfung back and front
+  useEffect(() => {
+    fetch('http://localhost:4000/exercises')
+      .then((res) => res.json())
+      .then((data) => {
```

Comment:
```suggestion
      .then((exerciseData) => {
```
data is a very generic name, try to find a more specific name